src/ResourceManagement/Network/Microsoft.Azure.Management.Network/Properties/AssemblyInfo.cs

```diff
@@ -7,8 +7,8 @@
 [assembly: AssemblyTitle("Microsoft Azure Network Management Library")]
 [assembly: AssemblyDescription("Provides Microsoft Azure Network management functions for managing the Microsoft Azure Network service.")]
 
-[assembly: AssemblyVersion("9.1.0.0")]
-[assembly: AssemblyFileVersion("9.1.0.0")]
+[assembly: AssemblyVersion("9.2.0.0")]
```

Comment:
@henry416 you can set `AssemblyVersion` to 9.0.0 and keep `AssemblyFileVersion` at 9.2.0.

`AssemblyFileVersion` should always match the version found in `project.json`, and `AssemblyVersion` shares only the major version of the `AssemblyFileVersion`